dns-policy-app/app/policy/management/commands/create_reviewer.py

```diff
@@ -22,15 +25,26 @@ def add_arguments(self, parser):
         """Define CLI arguments."""
         parser.add_argument('username', type=str, help='Username of the user')
         parser.add_argument('email', type=str, help='Email of the user')
+        parser.add_argument('--generate_password', action='store_true', help='Generate password')
 
     def handle(self, *args, **options):
         """Handle CLI invocation."""
-        # Check if there is any input available on sys.stdin
-        rlist, _, _ = select.select([sys.stdin], [], [], 0)
-        if rlist:
-            # Read password from STDIN
-            password = sys.stdin.readline().strip()
-        else:
+        password = ""
+
+        if options["generate_password"]:
+            password = ''.join(secrets.choice(string.ascii_letters + string.digits) for _ in range(16))
+
+        if password == "":
+            # Check if there is any input available on sys.stdin
+            rlist, _, _ = select.select([sys.stdin], [], [], 0)
+            if rlist:
+                # Read password from STDIN
+                password = sys.stdin.readline().strip()
+
+        if password == "":
+            password = os.getenv("PASSWORD", "")
```

Comment:
To avoid getting a different variable, wouldn't it help to add a prefix for PASSWORD?

Reply:
Changed